appveyor.yml

```diff
@@ -1,40 +1,30 @@
-# version here is only relevant for the AppVeyor CI, no need to adjust
-version: '{build}'
+version: '0.4.1.{build}'
 
 environment:
-  global:
-    PYTHON_LOC: "C:\\Miniconda36-x64"
+#  global:
+      # Example can be found here https://github.com/quantopian/zipline/blob/master/appveyor.yml
+#      ANACONDA_TOKEN: # not required anymore since it is configured at AppVeyor
+  matrix:
+    - PYTHON_LOC: "C:\\Miniconda37-x64"
+      PYTHON_VERSION: 3.7
+    - PYTHON_LOC: "C:\\Miniconda36-x64"
+      PYTHON_VERSION: 3.6
+    - PYTHON_LOC: "C:\\Miniconda35-x64"
+      PYTHON_VERSION: 3.5
 
 install:
-  - call %PYTHON_LOC%\Scripts\activate.bat
-  - conda config --set always_yes yes --set changeps1 no
-  - conda install -y conda-build constructor anaconda-client
+  - cmd: call %PYTHON_LOC%\Scripts\activate.bat
+  - cmd: conda config --set always_yes yes --set changeps1 no
+  - cmd: conda update -q conda
+  - cmd: conda info -a
 
 build_script:
-  # Test are disabled (--no-test): matplotlib fails, because no writable HOME directory
-  #                                AppVeyor has no HOME directory set.
-  - set MPLCONFIGDIR="."
-  #- conda build --old-build-string --no-test -c conda-forge -c defaults recipes\cate
-  - conda build --old-build-string --no-test -c conda-forge -c defaults recipes\cate-cli
-  #- conda build --old-build-string --no-test -c conda-forge -c defaults recipes\cate-util
-
-deploy_script:
-  # only publish artifacts on master branch when pushed (not from cron or PRs)
-  - ps: |
-      if ( $env:APPVEYOR_SCHEDULED_BUILD -eq "True" ) {
-          write-output "CRON build, won't deploy to anaconda"
-      } else {
-          $env:conda_upload = 'true'
-      }
-  - ps: |
-      if ([bool]$env:APPVEYOR_PULL_REQUEST_NUMBER) {
-          write-output "PULL REQUEST build, won't deploy to anaconda"
-      } else {
-          $env:conda_upload = 'true'
-      }
-  - ps: |
-      if ($env:APPVEYOR_REPO_BRANCH -eq 'master') {
-          $env:conda_upload = 'true'
-      }
-  - cmd: IF "%conda_upload%"=="true" anaconda -v -t %ANACONDA_TOKEN% upload %PYTHON_LOC%\conda-bld\win-64\cate*.tar.bz2 -u ccitools --force
-  - echo "Finished Artifact Deployment"
+  - cmd: conda env create --file environment-%PYTHON_VERSION%.yml
+  - cmd: call %PYTHON_LOC%\Scripts\activate.bat gridtools
+  - cmd: python setup.py install
+  - cmd: conda list
+  - cmd: py.test -v test
+  - cmd: call %PYTHON_LOC%\Scripts\activate.bat root
+  - cmd: conda install conda-build anaconda-client
+  #- cmd: conda build --old-build-string -c conda-forge -c anaconda recipe -m recipe/conda_build_config.yaml
+  #- cmd: anaconda -v -t %ANACONDA_TOKEN% upload "%PYTHON_LOC%\conda-bld\win-64\gridtools*.tar.bz2" -u cablab --force
```

Comment:
Please delete lines that are no longer needed. If they *may* be needed some day, add a comment saying so.